### ISSUE TITLE
Matrix/IRC Lack of proper moderation

### ISSUE THREAD

**Issue by dan-is-not-the-man** [monero-project/meta#899]
As this is creating issues within the community, never thought i would have to do this. But i think most people in the monero community matrix had enough have lost faith in some of the banning decisions made by some admin/moderators in monero matrix rooms and the direction its leading this community. I request that core intervene as there been massive inconsistencies/over reach with banning members of the community or lack of moderation.
 
1. banning someone for trolling when it was pointing out issues/discrepancies ie ofrnxmr

![events2](https://github.com/monero-project/meta/assets/142721047/ab2003db-5a62-437d-9955-017de1a31822)
![event3](https://github.com/monero-project/meta/assets/142721047/38c33672-c27d-4dd1-8395-3d315a76388d)
![events4](https://github.com/monero-project/meta/assets/142721047/34ddfbb2-97d1-45ba-8736-ae7eedbd903a)
![events5](https://github.com/monero-project/meta/assets/142721047/5d347001-5f3a-4be2-958c-664bced62c89)
![reso1](https://github.com/monero-project/meta/assets/142721047/1301e491-2064-4f70-926c-7bd9c7976e06)
![reso2](https://github.com/monero-project/meta/assets/142721047/499623d7-ff87-45d1-bbaa-17efcdb92193)
![reso3](https://github.com/monero-project/meta/assets/142721047/212f06cf-8924-43f7-b9ac-6bb6c5b7c564)
![reso4](https://github.com/monero-project/meta/assets/142721047/cc2ce02e-a167-4aa5-97a0-9f5761549a3c)
![reso5](https://github.com/monero-project/meta/assets/142721047/3c267cc2-399e-4a97-a0f3-ae9a2409e9a6)
![reso6](https://github.com/monero-project/meta/assets/142721047/a7bc9cff-37e7-4ed8-876f-ac25406872ae)
![reso7](https://github.com/monero-project/meta/assets/142721047/cc7afb08-1957-447a-85cb-f912da6e4edf)

2. threats of violence towards other members and ignoring.

https://matrix.org/legal/code-of-conduct/
```
Harassment includes, but is not limited to:
 - Threats of violence, both physical and psychological
 - Incitement of violence towards any individual, including encouraging a person to commit suicide or to engage in self-harm
 - Deliberate intimidation
```
# 3/10/23- threats of violence 

```
MajesticBank
I can't believe how some of ours community people behave, if one of their friends make worst ccs request ever for retroactive funding
16:31
MajesticBank
they come and write merge, merge
16:31
MajesticBank
If my friend wrote ccs like this, I would come to his door and kick him in the head with the keyboard
16:32
MajesticBank
Tell him to stop disrespecting me, him and the ccs fund
16:34
MajesticBank
For me there is 1% chance for k​ayabanerve proposal to be accepted AS-IS
16:38
MajesticBank
specially since 0 suggestions were consider and feedback from the community deemed as 'not-necessary'
```

### https://libera.monerologs.net/monero-community/20230903#c276461

# 4/09/23
```
10:25
m-relay
<k​ayabanerve:matrix.org> 1) I'm more receptive to feedback when it's not accompanied with a comment I should have my head bashed in
```
https://libera.monerologs.net/monero-community/20230904#c277318

3. excessive ban length for minor reasons
![events15](https://github.com/monero-project/meta/assets/142721047/18a034d0-43f5-4c35-951e-3601ebde0a5a)
![reso7](https://github.com/monero-project/meta/assets/142721047/e8f6118d-eb6f-429a-9739-f2328deb4639)

4. Trolling

# 4/10/23- trolling

![IMG_8761](https://github.com/monero-project/meta/assets/142721047/8201be93-54eb-4893-9e57-7822bc2afebd)
![IMG_8759](https://github.com/monero-project/meta/assets/142721047/791b1147-1d8d-4eee-b774-2287a980ee36)
![IMG_8757](https://github.com/monero-project/meta/assets/142721047/2b7e826d-c164-45ea-8c43-6584032b997e)

![IMG_8685](https://github.com/monero-project/meta/assets/142721047/945dd1cc-5295-4c6c-ac3a-9ac98a82fd60)
![geonic image1](https://github.com/monero-project/meta/assets/142721047/944966e3-56f7-431e-b261-817e78f129b1)

```
07:15
m-relay
<1​23bob123:matrix.org> miss use email been what 3 weeks now
07:16
m-relay
<1​23bob123:matrix.org> trying to rip off community members what 2 months since monerokon?
07:18
geonic
has the troll farm officially moved from -events to -community now?
07:19
m-relay
<1​23bob123:matrix.org> thank you for making my point
07:19
m-relay
<s​iren:kernal.eu> Broken record
07:19
m-relay
<c​trej:matrix.org> "accidentally"
07:20
geonic
have fun fellas
07:20
m-relay
<c​trej:matrix.org> The accidental vote stayed
07:23
m-relay
<1​23bob123:matrix.org> old guard need new blood i guess
07:26
geonic
new blood testing positive for leukemia :/
07:27
geonic
let me just point out that using a CCS proposal you hate as reasoning to accept literally *anything* shows that you’re operating in bad faith
07:29
geonic
“what I want more for Monero is what I hate” so pls merge ok tx
07:34
geonic
and yeah you’ve literally been around for 5 minutes: repo.getmonero.org/monero-project/c…als/-/merge_requests/371#note_20457
07:35
geonic
btw, why are you referring to yourself in the third person here? “ct has been around..”
07:37
geonic
too many personalities to juggle?
```
### https://libera.monerologs.net/monero-community/20230904

I think the following should be discussed in next community meeting
 
 1. mods/admins need to be same across all monero rooms
 2. clearer rules for bannable offense. 
 3. cause this is community based that the community should vote and decide who should be moderators/admins.
 4. No member should be admin/mod if they have corporate interests, to allow for impartiality
 5. Not banning people from "Moderation Resolution room" how do you appeal or rebut the allegations? 
 
![IMG_8752](https://github.com/monero-project/meta/assets/142721047/96c9d3a1-5e7d-4e0d-a332-fd4290568971)

 
 7. All Rooms bridged to IRC for logging

**Comment by nahuhh** [comment]
Today (Sat Dec 23, 2023), MoneroKon aka Twisted Edwards (an Non-Profit organization in the making) shall propose a new "MoneroKon Chat Moneration Policy".

The policy details rules and expectations of MoneroKon (org). yellow, red card, perm bans, etc.

That is illogical and ridiculous because:
- MoneroKon _uses_ the #monero-events room. 
- MoneroKon is a company, and companies don't speak for Monero
- #monero-events is a Monero channel, not a Twisted Edwards channel

its baffling to think that a corporate entity can dictate the rules of the rooms they are ALLOWED to use. A MoneroKon Chat Policy can only apply to MoneroKon discussions. MoneroKon(TE) cannot silence defcon, monerotopia or ofrntopiakon discussions. 

Imagine if Monerotopia wanted to perm ban people from #monero-events, for breaking monerotopia's internal policies 

#update dec 24, 2023: 

@SamsungGalaxyPlayer [SGP] 
- is Admin of monero-events
- is Admin of almost every monero-* matrix room
- recently became deeply involved in Monerokon operations

At the "MoneroKon Planning Meeting", "Monerokon" voted to enact a new "Monerokon chat moderation policy" from github.com/monerokon/meta.
SGP has since **decided** that simply being an Admin, gives him the authority to apply _moderation_ policies of companies he works for to rooms he administers for monero.social. 

I've left the channel (monero-events), as i have no interest in entertaining such nonsense.

logic:
1. Not "monerokon", but _monero-events_ has a meeting and votes on a _monero-events_ moderation policy or
2. Monerokon has a room where they can enact policies, such as #monerokon:monero.social

illogical nonsense:
1. **Monerokon** votes for a **monerokon** policy, and sgp applies it to **monero-events** 🤦‍♂️ (and any other room that he chooses)
2. When another event needs to use the room "we can **discuss** using libera/monero.social policies instead of monerokon's policy" says sgp 🤡


My recommendation? 
Admins, administrate
Mods, moderate

People who work for companies work alongside the feds (moonstone research) should not be in control of monero discourse, particularly when the moderation you _see_ from sgp is more often than not, personal. 

tldr: sgp shouldnt be moderating anything aside from his emotions. "does anyone mind if i ban endogenic??", "im keeping rotten banned because of this 2 year old tweet"

**Comment by nahuhh** [comment]
You guessed it: ofrnxmr problems again

![b60yzjxclcu97ag3.jpg](https://github.com/monero-project/meta/assets/50635951/b17433f7-9ed1-4426-b596-799a8122cd7f)

> <@ofrnxmr:monero.social> .. still wondering what rooms i was banned in

> (erc) monero-website (monero-site) and monero-translations.

- I wasn't a member monero-site and havent been since erc kicked me many months ago re: he refused to upload diffs when adding paper wallet generator. He got mad after luigi told him to add the diffs and i rubbed it in his face
- i was in translations, but dont think ive ever commented more than a few messages over the years 


his reason for banning me today? 
A user couldn't join the Haveno matrix room.
The reason being, the room was hosted on erc's haveno.network homeserver and was never properly federated to monero.social or matrix.org. the main address is still haveno.network.

he cant handle being incorrect, so as expected of erc, he played the victim and came to my dm's to let me know hes banned me. (left before i could reply, then cried foul that i posted the screenshot and replied in public).

Anyway, erc is another mod who lets emotions dictate his actions and inability to accept valid criticism without claiming to be threatened.
